src/madsci_workcell_manager/madsci/workcell_manager/workcell_engine.py

```diff
@@ -48,8 +49,12 @@ def __init__(
         scheduler_module = importlib.import_module(self.definition.config.scheduler)
         self.scheduler = scheduler_module.Scheduler(self.definition, self.state_handler)
         self.data_client = DataClient(self.definition.config.data_server_url)
+        self.resource_client = ResourceClient(
+            self.definition.config.resource_server_url
+        )
+        default_logger.log_info(self.data_client.url)
```

Comment:
We should create a logger for the workcell engine, rather than using the default.